docs/adrs/0004-field-for-institution-information.md

```diff
@@ -0,0 +1,119 @@
+      * **TIMDEX field mapping**: [`publisher` -> `publication_information`](https://github.com/MITLibraries/transmogrifier/blob/main/transmogrifier/sources/xml/datacite.py#L249-L256)
+      * **Example value(s):** As of this writing, the TIMDEX records from JPAL most likely all read "Harvard Dataverse" for this field; records from Zenodo most likely all read "Zenodo".
+
+   **Transformer:** DspaceDim
+   **Source(s):** Woodshole Open Access Server (whoas)
+   * [`<dim:field mdschema="dc" element="publisher"`>](https://www.dublincore.org/specifications/dublin-core/dcmi-terms/#http://purl.org/dc/terms/publisher): An entity responsible for making the resource available.
+      * **TIMDEX field mapping:** [`publisher` -> `publication_information`](https://github.com/MITLibraries/transmogrifier/blob/main/transmogrifier/sources/xml/dspace_dim.py#L204-L207)
+      * **Example value(s):** As of this writing, the TIMDEX records from WHOAS most likely all read "Woods Hole Oceanographic Institution" for this field.
+
+   **Transformer:** DspaceMets (DSpace@MIT)
+   **Source(s):** Dspace@MIT (dspace)
+   * [`mods:publisher`](https://www.loc.gov/standards/mods/userguide/origininfo.html#publisher): The name of the entity that published, printed, distributed, released, issued, or produced the resource.
+      * **TIMDEX field mapping:** [`publisher` -> `publication_information`](https://github.com/MITLibraries/transmogrifier/blob/main/transmogrifier/sources/xml/dspace_mets.py#L150-L152)
+      * **Example value(s):** Massachusetts Institute of Technology, Springer Science and Business Media LLC, Public Library of Science (PLoS)
+
+   
+   **Transformer:** [Ead](https://www.loc.gov/ead/tglib/appendix_d.html)
+   **Source(s):** MIT ArchivesSpace (aspace)
+   * [`<publicationstmt><publisher>`|`<bibliography><bibref><imprint><publisher>`](https://www.loc.gov/ead/tglib/elements/publisher.html): When used in the publication statement, the name of the party responsible for issuing or distributing the encoded finding aid. Often this party is the same corporate body identified in the `<repository>` element in the finding aid. When used in a Bibliographic Reference <bibref>, the name of the party issuing a monograph or other bibliographic work cited in the finding aid.
+   * [`repository`](https://www.loc.gov/ead/tglib/elements/repository.html): The institution or agency responsible for providing intellectual access to the materials being described. Although the repository providing intellectual access usually also has physical custody over the materials, this is not always the case.  When it is clear that the physical custodian does not provide intellectual access, use <physloc> to identify the custodian and <repository> to designate the intellectual caretaker. When a distinction cannot be made, assume that the custodian of the physical objects also provides intellectual access to them and should be recognized as the <repository>.
```

Comment:
Wonder if mapping EAD `repository` to this new `institution/provider` field might make sense?

Reply:
@ghukill @JPrevost 

Would it also be worth to add a mapping from the `publisher` field to the `provider` field for:

1. **Datacite:** The TIMDEX records from JPAL most likely all read "Harvard Dataverse" for this field; records from Zenodo most likely all read "Zenodo"
2. **DspaceDim:** The TIMDEX records from WHOAS most likely all read "Woods Hole Oceanographic Institution" for this field.
3. **SpringshareOaiDc:** For LibGuides, this value will always read "MIT Libraries"
?